modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -19,9 +19,34 @@ Common scenarios for using the back up and restore process include:
 --
 Do not use the back up and restore process to create concurrent running instances of Runtime Fabric. In other words, do not back up Runtime Fabric A, use that back up to create a new Runtime Fabric B while Runtime Fabric A is still active, and attempt to run them concurrently. This creates application deployment issues.
 --
-== The Back Up and Restore Process Does Not Affect the Control Plane
 
-The Runtime Fabric back up and restore process affects only the Runtime Fabric cluster, and it does not involve the control plane. Any changes you make after you perform a backup will not be present if you restore the cluster from that backup. For example, if you perform a backup, delete an application, and then restore from that backup, the application you deleted will be recreated in the cluster, but it will be marked as deleted in Runtime Manager. As a result the application's status will show as *applying* in Runtime Manager.
+== What Gets Backed Up
+
+The Runtime Fabric back up process affects only the Runtime Fabric cluster. The back up process does not back up data from the control plane.
+
+The following is backed up:
+
+* Runtime Fabric cluster details, such as controller and worker node configurations
+* Cluster-level secure properties 
+* All data in the Runtime Fabric and application namespace, such as data about deployments, services and service accounts, secrets, ingress resources, role bindings, cluster role configurations, and ConfigMap settings
+* The current `rtfctl` version
+* Application details including:
+** Application configurations for pods, replicasets, application metadata, and templates
+** Application properties configured in Runtime Manager
+* Data from other Mulesoft configurations, including:
+** Object Store
+** Edge
+** Anypoint Monitoring and external logforwarders
+
+== What Does Not Get Backed Up
+
+* Data about application state
+* Data from Ops Center
+* Data from the control plane
+
+=== Changes Made After Performing a Backup Are Not Restored
+
+Any changes you make after you perform a backup will not be present if you restore the cluster from that backup. For example, if you perform a backup, delete an application, and then restore from that backup, the application you deleted will be recreated in the cluster, but it will be marked as deleted in Runtime Manager. As a result the application's status will show as *applying* in Runtime Manager.
```

Comment:
suggest:
... the application you deleted is recreated in the cluster, but it is marked as deleted in Runtime Manager. As a result, the application's status s shows as *applying* in Runtime Manager.